core/src/main/java/org/elasticsearch/rest/action/support/RestActions.java

```diff
@@ -64,18 +73,24 @@ public static long parseVersion(RestRequest request, long defaultVersion) {
     }
 
     static final class Fields {
+        static final String _NODES = "_nodes";
```

Comment:
Nit: I'd just use the string rather than make a constant. We are slowly removing these objects.